Staatliche Berufsschule I Bayreuth

jhoesl opened the pull request:
#### Institution/School Name 
Staatliche Berufsschule I Bayreuth

#### Instiution/School Website
www.bs1-bt.de

#### Email Users

*Please place an x between the square brackets if yes*
- [x] Students
- [x] Academic/Teaching Staff
- [ ] Other/Support Staff
- [ ] Alumni

#### Education Levels

*Please place an x between the square brackets if yes*

- [ ] Post-graduate research
- [ ] Graduate School
- [ ] College (University) or Undergraduate School or equivalent
- [x] Community College or equivalent
- [x] High School or equivalent
- [ ] Elementary School or equivalent

[bs1-bt.txt](https://github.com/leereilly/swot/files/3206181/bs1-bt.txt)

daydreamsj123 commented:
[file.zip](https://github.com/leereilly/swot/files/3279355/file.zip)
please,add my school,Thank you

TimeBoat commented:
Add HeHai University in China

3than7 commented:
[file.zip](https://github.com/leereilly/swot/files/3457494/file.zip)

please,add my College,Thank you

chenzheng950322 commented:
please,add my school,Thank you
[file.zip](https://github.com/leereilly/swot/files/3500577/file.zip)

janbrookshoffmann commented:
Could you also please add my school ? (KBZSG) 
[kbzsg.txt](https://github.com/leereilly/swot/files/3531512/kbzsg.txt)

DXR397 commented:
加入山东英才学院 
[sdycxy.txt](https://github.com/leereilly/swot/files/3539446/sdycxy.txt)

DXR397 commented:
Join Shandong Yingcai College
[sdycxy.txt](https://github.com/leereilly/swot/files/3539453/sdycxy.txt)

qaqaqwqe commented:
please add my school.

thanks.
[file.zip](https://github.com/leereilly/swot/files/3561663/file.zip)

crislizame commented:
[file.zip](https://github.com/leereilly/swot/files/3597878/file.zip)
Please add my School, Ecuator High School, name "Augusto Mendoza Moreira" Ministerio de Educacion. educacion.gob.ec

attmaxess commented:
[file.zip](https://github.com/leereilly/swot/files/3598582/file.zip)
Here is my school. Virginia Western Community College. Please add.

Ishikawwwa commented:
Please add my school 1329
[edu.txt](https://github.com/leereilly/swot/files/3626151/edu.txt)

CLOSED-EX-TITANDEV commented:
Hello
Can you add my school, Thanks you

[file.zip](https://github.com/leereilly/swot/files/3658838/file.zip)

timleg002 commented:
Please add my school
[file.zip](https://github.com/leereilly/swot/files/3665885/file.zip)

tywish-dev commented:
please add my school!
Kozmos Yazılım Anadolu Lisesi Türkiye // Kozmos Software High School in Turkey
[file.zip](https://github.com/leereilly/swot/files/3699263/file.zip)

Daurenbekuly commented:
please add my university Kazakh University of Economics of Finance and International Trade
https://kuef.kz/ru/
alibi@bk.ru
Undergraduate
[file.zip](https://github.com/leereilly/swot/files/3753885/file.zip)

shohruhsadullayev commented:
[file.zip](https://github.com/leereilly/swot/files/3763712/file.zip)
please, add my university

IMICEEBADDGER commented:
[students.txt](https://github.com/leereilly/swot/files/3812306/students.txt)

please add my school thanks

richlysakowski commented:
[ntai.txt](https://github.com/leereilly/swot/files/3855276/ntai.txt)
Network Technology Academy Institute in Malden, Massachusetts.  We teach networking, computer science, systems administration, data science, databases, and other technical courses lasting from 6 months to 2 years.

guorouda commented:
[ccc.txt](https://github.com/leereilly/swot/files/3885481/ccc.txt)
please add my college

richlysakowski commented:
How do I unsubscribe to this forum?  I keep getting email notifications anytime someone gets added to this list.   I need to stop getting notified.

LukaTarkhn commented:
[file.zip](https://github.com/leereilly/swot/files/3920466/file.zip)
Science education structure in georgia add please

Alejandroregue commented:
Agrega mi escuela por favor
[file.zip](https://github.com/leereilly/swot/files/3924130/file.zip)

ahmadalnaib commented:
Hi please add my uni 
https://kiron.ngo
[file.zip](https://github.com/leereilly/swot/files/3940688/file.zip)

lib/domains/ngo/kiron.txt

ahmedhabeeb73 commented:
lib/domains/ngo/kiron.txt

jaredlew commented:
Hallo,

I added Blue Ridge Community College to the list.
Thank you for merging in advance.

[file.zip](https://github.com/leereilly/swot/files/3968628/file.zip)

TomaszLuszczuk commented:
Hello,

I added Wyższa Szkoła Zarządzania Ochroną Pracy w Katowicach to the list.
Thank you for merging in advance.

[file.zip](https://github.com/leereilly/swot/files/3973120/file.zip)

snipersix6 commented:
[file.zip](https://github.com/leereilly/swot/files/3976638/file.zip)
Hi,
pls add my College MVSO.
Website: mvso.cz/en/
TNX

CQULucas commented:
add Chongqing University
[cqu.txt](https://github.com/leereilly/swot/files/4000488/cqu.txt)

Aranccar commented:
Please add University of Central Asia
[ucentralasia.txt](https://github.com/leereilly/swot/files/4015908/ucentralasia.txt)

abrahamhadaf commented:
Please add my school

[file.zip](https://github.com/leereilly/swot/files/4043017/file.zip)

weihuayao commented:
please,add my school,Thank you。Jiangxi University of Finance and Economics
[file.zip](https://github.com/leereilly/swot/files/4049358/file.zip)

yangchangyong0 commented:
重庆大学
Chongqing University
Here is my school. Please add.
[file.zip](https://github.com/leereilly/swot/files/4050166/file.zip)

**Instiution/School Website**
https://www.cqu.edu.cn/

Email Users
Please place an x between the square brackets if yes

 Students
 Academic/Teaching Staff


 Post-graduate research
 Graduate School
 College (University) or Undergraduate School or equivalent
 Community College or equivalent

Pidan-Q commented:
[file.zip](https://github.com/leereilly/swot/files/4053278/file.zip)
please,add my school,Thank you Chifeng University

cha1324406184 commented:
[file.zip](https://github.com/leereilly/swot/files/4063547/file.zip)
please,add my school,Thank you

songjinhe commented:
Add  Northern Virginia Community College
[https://www.nvcc.edu/programs/it/emerging.html](url)
[vccs.txt](https://github.com/leereilly/swot/files/4068906/vccs.txt)

ghost commented:
Please add my school:

LaSalle Academy

URL:  lasalle-academy.org
email domain:  lasalle-academy.org

[file.zip](https://github.com/leereilly/swot/files/4079956/file.zip)

cvjulian commented:
please,add my school,Thank you
[file.zip](https://github.com/leereilly/swot/files/4081545/file.zip)

rahulchaudhary-research commented:
[file.zip](https://github.com/leereilly/swot/files/4105145/file.zip)
Please add

Lichtdino commented:
Can you add my school please. Github would let me do a pull request i don't know why.
[tamcc.txt](https://github.com/leereilly/swot/files/4113101/tamcc.txt)

Michael-chko commented:
please add this school in russia
[edu.txt](https://github.com/leereilly/swot/files/4113237/edu.txt)

lazesmash commented:
please add my school
[file (1).zip](https://github.com/leereilly/swot/files/4122237/file.1.zip)

emmanuelouedraogo76 commented:
[file(1).zip](https://github.com/leereilly/swot/files/4193628/file.1.zip)
please add my university

RunerJone commented:
[file.zip](https://github.com/leereilly/swot/files/4204702/file.zip)
please add my university

emmanuelouedraogo76 commented:
[file.zip](https://github.com/leereilly/swot/files/4207642/file.zip)

quyetict commented:
Please help me to add the attach file to the link below.
https://github.com/JetBrains/swot/tree/master/lib/domains/vn/edu

[utehy.txt](https://github.com/leereilly/swot/files/4255794/utehy.txt)

emmanuelouedraogo76 commented:
Please add my school

eduard-lt commented:
Hi please add my college.
[student.txt](https://github.com/leereilly/swot/files/4302854/student.txt)

QViet5091 commented:
[file.zip](https://github.com/leereilly/swot/files/4332250/file.zip)
please,add my school,Thank you

linhktmt commented:
Please add my university
Thainguyen University of Technology, Vietnam
Instiution/School Website
http://en.tnut.edu.vn/
[file.zip](https://github.com/leereilly/swot/files/4366918/file.zip)

LezhninaYu commented:
Please, add my university in the accepted list.
http://агасу.рф/
Astrakhan state university of architecture and civil engineering. We have bachelor's and master's education program in IT sfere.
At the present time we have 3 relevant domens. There are *@aucu.ru, *@agasu.ru, *@asuace.ru
[file.zip](https://github.com/leereilly/swot/files/4402650/file.zip)

natran23 commented:
please,add my school,Thank you

[file.zip](https://github.com/leereilly/swot/files/4405375/file.zip)

twobrassiere reviewed (commented):
[mail.txt](https://github.com/leereilly/swot/files/4428704/mail.txt)

Ker-ri-shan commented:
please,add my school,Thank you .
[file.zip](https://github.com/leereilly/swot/files/4433086/file.zip)

AbanobNabil commented:
Institution/School name

Thebes Higher Institute of Computer and Management Sciences 
Maadi , Cairo , Egypt 

Email Users

 Students
Education Levels
Graduate School

[thebes.txt](https://github.com/leereilly/swot/files/4445968/thebes.txt)

ArjunParthasarathy commented:
Hi,
Can you please add my school to the list?

[csus.txt](https://github.com/leereilly/swot/files/4448695/csus.txt)

We are a high school called Crystal Springs Uplands School based in Hillsborough, California. Our home website is [csus.org](url)

Among other STEM courses, our school offers computer science and advanced statistics classes. Using JetBrains products thus far has greatly accelerated our school's ability to teach effectively and for the students to learn well. We hope that accessing the greater suite of JetBrains products will inspire many students at our school to go further and pursue careers in Computer Science.

linhqhta commented:
Please add edu domain nexta.edu.vn
[nexta.txt](https://github.com/leereilly/swot/files/4459635/nexta.txt)

phungduyvu commented:
please add my school! buh.edu.vn
[file.zip](https://github.com/leereilly/swot/files/4507341/file.zip)

prysousa commented:
[file.zip](https://github.com/leereilly/swot/files/4516916/file.zip)

Please add Centro Universitário Fametro

maksymkhodakov commented:
> #### Institution/School Name
> Staatliche Berufsschule I Bayreuth
> 
> #### Instiution/School Website
> [www.bs1-bt.de](http://www.bs1-bt.de)
> 
> #### Email Users
> _Please place an x between the square brackets if yes_
> 
> * [x]  Students
> * [x]  Academic/Teaching Staff
> * [ ]  Other/Support Staff
> * [ ]  Alumni
> 
> #### Education Levels
> _Please place an x between the square brackets if yes_
> 
> * [ ]  Post-graduate research
> * [ ]  Graduate School
> * [ ]  College (University) or Undergraduate School or equivalent
> * [x]  Community College or equivalent
> * [x]  High School or equivalent
> * [ ]  Elementary School or equivalent
> 
> [bs1-bt.txt](https://github.com/leereilly/swot/files/3206181/bs1-bt.txt)

[file.zip](https://github.com/leereilly/swot/files/4522940/file.zip)
Add my school pleeeeeeeeeeeeeeeeeeease)

maksymkhodakov commented:
Please add my favourite school in kyiv)
[file.zip](https://github.com/leereilly/swot/files/4523020/file.zip)

erick-code commented:
> Institution/School Name
> Colegio de Estudios Científicos y Tecnológicos del Estado de México
> Instiution/School Website
> http://cecytem.edomex.gob.mx/
> Email Users
>erick.luna.dx4@soycecytem.mx
> Please place an x between the square brackets if yes
> 
> 
>   
>     
>    Students
> 
>   
>     
>    Academic/Teaching Staff
> 
>   
>     
>    Other/Support Staff
> 
>   
>     
>    Alumno
> 
> Education Levels
> Please place an x between the square brackets if yes
> 
> 
>   
>     
>    Post-graduate research
> 
>   
>     
>    Graduate School
> 
>   
>     
>    College (University) or Undergraduate School or equivalent
> 
>   
>     
>    Community College or equivalent
> 
>   
>     
>    High School or equivalent
> 
>   
>     
>    Elementary School or equivalent
> 
[soycecytem.txt](https://github.com/leereilly/swot/files/4525917/soycecytem.txt)

> soycecytem.txt
Pleas can you add my school ?

rbjavi commented:
Hi. Could you please add my high school? 
[file.zip](https://github.com/leereilly/swot/files/4529950/file.zip)

Thank you.

hasanuygur commented:
[file.zip](https://github.com/leereilly/swot/files/4568942/file.zip)
pls add my school, thank you.

tzustu commented:
[lib.zip](https://github.com/leereilly/swot/files/4570736/lib.zip)
Please add Tzu Chi University of Science and Technology
[tcust.txt](https://github.com/leereilly/swot/files/4570725/tcust.txt)

alexfoderaro commented:
[file.zip](https://github.com/leereilly/swot/files/4574902/file.zip)

It is one of the most evaluated high school in Italy... 
Please add it

SabadModi commented:
Please add my school: 
Gems Modern Academy
https://www.gemsmodernacademy-dubai.com/

[file.zip](https://github.com/leereilly/swot/files/4623930/file.zip)
[gemselearning.txt](https://github.com/leereilly/swot/files/4623932/gemselearning.txt)

vietvv commented:
Please add my college caodangcntt.edu.vn to repository 

My College is: Professional IT College
Website: https://caodangcntt.edu.vn/
In Da Nang, Viet Nam

We teach programming Java, PHP so I need license InteliJ IDEA for our student. Would you please review for us.
Thank You.

[file.zip](https://github.com/leereilly/swot/files/4626638/file.zip)
[caodangcntt.txt](https://github.com/leereilly/swot/files/4626641/caodangcntt.txt)

vlachosaqua commented:
Please add my school, thanks. 
Website: http://sh5.do.am/
[file.zip](https://github.com/leereilly/swot/files/4643623/file.zip)

Daedalus-Zhang commented:
[file.zip](https://github.com/leereilly/swot/files/4674032/file.zip)
please add my school
STPA (south texas preparatory academy)

mrschuetze commented:
Please add my school
[file.zip](https://github.com/leereilly/swot/files/4676917/file.zip)

AlterK-Git commented:
Hi, can you add my school please,
(CFI Formations)

thanks
[file.zip](https://github.com/leereilly/swot/files/4694416/file.zip)

Vutha commented:
[file.zip](https://github.com/leereilly/swot/files/4697718/file.zip)
Institution/School Name
សាលាជាតិរដ្ឋបាលមូលដ្ឋាន
National School of Local Administration (NASLA)

Instiution/School Website
www.ncdd.gov.kh
https://bit.ly/3ery16R

 Students
 Academic/Teaching Staff
 Other/Support Staff
 
- National School of Local Administration (NASLA) is under the managing of National Committee for Sub-National Development (NCDD)
- We provide training and class for sub-national officer related to ICT and Database System
- Please help to add our domain
Thank you very much

abelogo commented:
Hi there,

Could you please add my school?

Thanks a lot

[file.zip](https://github.com/leereilly/swot/files/4725984/file.zip)

philipto committed:
Rename lib/domains/huddinge/edu.txt to lib/domains/se/huddinge/edu.txt

philipto committed:
Merge pull request #31928 from fkaurrany/master

Adding new domain ibu.ca and myibu.ca

philipto committed:
Merge pull request #32463 from SuperRedstoneYT/Add-stu.broxbourne

Pull request accepted.

philipto committed:
Merge pull request #32461 from BenjaminHesias/master

Pull request accepted.

philipto committed:
Merge pull request #32448 from RobertLQuigley/master

Pull request accepted.

philipto committed:
Merge pull request #32314 from alanolguin33/master

Pull request accepted.

philipto committed:
Merge pull request #32029 from hislost/master

Pull request accepted.

philipto committed:
Merge pull request #31836 from tanishq-tron/patch-1

Pull request accepted.

philipto committed:
Merge pull request #31809 from Adam-M-Mahmoud/master

Talae Al Amal Language School

philipto committed:
Merge pull request #31848 from AdityaTile04/master

Add matoshri.edu.in domain for Matoshri College of Engineering & Rese…

philipto committed:
Merge pull request #31871 from ekasatriaap/ekasatriaap-jetbrains-merangin

add files via upload universitasmerangin.txt

philipto committed:
Merge pull request #31883 from AlexisOzaetaM/itd

feat: create itdelicias.txt

philipto committed:
Merge pull request #31885 from seungwon2jun/seungwon2jun-patch-1

Seungwon2jun patch 1

philipto committed:
Merge pull request #31951 from lucaschoeneberg/master

Add artandtech.space for ARTandTECH.space e.V. (Rheine, DE)

philipto committed:
Merge pull request #32003 from WebDclassified/master

Add files via upload

philipto committed:
Merge pull request #32041 from RevanRaoK/master

Adding KMIT domain

philipto committed:
Merge pull request #32057 from Marc-Blanco/master

Ies Cap de llevant

philipto committed:
Merge pull request #32068 from jfbernalp/patch-1

Add Fundación Universitaria Cafam (unicafam.edu.co)

philipto committed:
Merge pull request #32086 from adriana71/patch-1

Create uaq.txt to add the domain of  Universidad Autónoma de Querétaro

philipto committed:
Merge pull request #32234 from AlexanderBarnes0613/master

Please add my school's domain name

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

masouto94 committed:
Create unsam-bue.txt

Commit:
Added ktgg.txt

Commit:
Added ktgg english name

scentis committed:
Create estudiantes.txt

www.ucnl.edu.mx/

0xviny committed:
Add files via upload

mrrobotoo committed:
Add files via upload

r2r-troberts committed:
Add Clearview Regional High School

aobiji committed:
Create aobiji

domain add aobiji.ac.jp

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #32486 from aobiji/master

Pull request accepted.

philipto committed:
Merge pull request #32483 from r2r-troberts/add-clearview

Pull request accepted.

philipto committed:
Merge pull request #32482 from mrrobotoo/master

Pull request accepted.

hareesh08 committed:
Added SRM Easwari Engineering College

Added SRM Easwari Engineering College

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #32471 from moniquet-dot-a/master

Pull request accepted.

philipto committed:
Merge pull request #32467 from masouto94/master

Pull request accepted.

JustEthanCodes committed:
Add files via upload

armanvnv committed:
Add domain dsu.edu.in.

ritika-9 committed:
Add Shri Ram Murti Smarak College of Engineering and Technology (srmscet.edu)

ritika-9 committed:
Add Shri Ram Murti Smarak College of Engineering and Technology (srmscet.edu)

PecceG2 committed:
Create cts.txt

PecceG2 committed:
Merge pull request #1 from PecceG2/PecceG2-patch-1

Create cts.txt
long-term related study plan: https://cts.edu.ar/plan_de_estudio.html

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

professorwade committed:
Create centralvirginia.txt

College URL: centralvirginia.edu
CS Program: https://centralvirginia.edu/programs-classes/science-technology-engineering-math/computer-science

philipto committed:
Merge pull request #32499 from professorwade/master

Pull request accepted.

philipto committed:
Merge pull request #32497 from PecceG2/master

Pull request accepted.

philipto committed:
Merge pull request #32493 from ritika-9/master

Pull request accepted.

philipto committed:
Merge pull request #32491 from armanvnv/master

Pull request accepted.

philipto committed:
Merge pull request #32487 from hareesh08/master

Pull request accepted.

philipto committed:
Cleanup

philipto committed:
Merge remote-tracking branch 'origin/master'

SistemasSEAUMN committed:
Create crg.txt

arlopeza73 committed:
Create pucesa.txt

Creacion de la PUCESA

tgs-johnny committed:
add trinity grammar kew

philipto committed:
Merge pull request #32525 from tgs-johnny/master

Pull request accepted.

philipto committed:
Merge pull request #32521 from arlopeza73/master

Pull request accepted.

brandonhoeppner committed:
Delete file.zip

martincraviotto committed:
Add new txt with information : Education

I added a new txt document with information about the University (Instituto Universitario Vucetich).

BryanCardiet committed:
Create idealschool.txt

https://idealschool.education

shubhimathur069 committed:
Add Narayana Business School

Pancaakeman committed:
Added the HRUC colleges

mukarramcodex committed:
Create RPI.txt Thanks

philipto committed:
Merge pull request #32566 from mukarramcodex/master

Pull request accepted.

philipto committed:
Merge pull request #32555 from Pancaakeman/master

Pull request accepted.

philipto committed:
Merge pull request #32548 from shubhimathur069/master

Pull request accepted.

philipto committed:
Merge pull request #32545 from BryanCardiet/master

Pull request accepted.

philipto committed:
Merge pull request #32543 from martincraviotto/master

Pull request accepted.

jk0re committed:
Add IIS Confalonieri - De Chirico to Italian educational domains

philipto committed:
Merge pull request #32572 from jk0re/add-confalonieri-school

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Merge pull request #32125 from ammarabbasi/master

 Educational Subscription

philipto committed:
Merge pull request #32131 from robjo82/master

Create insa-hdf.txt

philipto committed:
Merge pull request #32158 from bahrulupjb/master

Add upjb ( Universitas PGRI Jombang)

philipto committed:
Merge pull request #32185 from zephyr-profile/master

add siu.ac.th, Shinawatra University

philipto committed:
Merge pull request #32255 from MihajloMirkovic/master

Add files via upload

haryandb committed:
Create ummuba.txt

volrriti committed:
Add files via upload

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

philipto committed:
Fraud

VinceSuela committed:
Add files via upload

philipto committed:
Merge pull request #32603 from VinceSuela/master

Pull request accepted.

philipto committed:
Merge pull request #32602 from volrriti/master

Pull request accepted.

philipto committed:
Merge pull request #32587 from haryandb/master

Pull request accepted.

pabloricoeasda committed:
Students and teachers of Higher Institute of Artistic Education Comunidad Valenciana, Spain

MironovAlexanderJR committed:
Ritsumeikan Asia Pacific University

M-i-h-a-i committed:
AvramIancuCluj

philipto committed:
Merge pull request #32611 from M-i-h-a-i/master

Pull request accepted.

philipto committed:
Merge pull request #32607 from MironovAlexanderJR/master

Pull request accepted.

philipto committed:
Merge pull request #32606 from pabloricoeasda/master

Pull request accepted.

bpluscDev committed:
Create institutouniversitarioibr.txt

IDontLikeGHIJustNeedToSubmitAPr committed:
:3

add fulbrook.school

Ari122003 committed:
Create tint.txt

rahuldora71 committed:
Geeta university Added 

Add Geeta university

danlopezmm committed:
Create universidad-uic.txt

basim-zic committed:
Add Zikura International College

basim-zic committed:
Merge branch 'master' of https://github.com/basim-zic/swot

philipto committed:
Merge pull request #32644 from basim-zic/master

Pull request accepted.

philipto committed:
Merge pull request #32641 from danlopezmm/master

Pull request accepted.

philipto committed:
Merge pull request #32640 from rahuldora71/master

Pull request accepted.

philipto committed:
Merge pull request #32626 from bpluscDev/master

Pull request accepted.

philipto committed:
Merge pull request #32625 from Ari122003/patch-1

Pull request accepted.

philipto committed:
Merge pull request #32623 from IDontLikeGHIJustNeedToSubmitAPr/patch

Pull request accepted.

philipto committed:
The University of Computer Studies,Yangon

philipto committed:
Merge remote-tracking branch 'origin/master'

sonbui00 committed:
Add HANOI VOCATIONAL COLLEGE OF TECHNOLOGY entry

Commit:
Vietnam Japan University

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Fraud

philipto committed:
Delete grantcareer.txt

philipto committed:
Merge pull request #32516 from brandonhoeppner/master

No Clue

philipto committed:
Merge pull request #32272 from Genilsonsdg/master

feat(lib/domains): add new udu - uncisal

philipto committed:
Merge pull request #32325 from sontus/master

Please Add A.M. Technical Training Academy

philipto committed:
Merge pull request #32380 from Miichelle-27/master

Add files via upload

philipto committed:
Merge pull request #32407 from beloniara/master

Added National University - East Ortigas student email domain

philipto committed:
Merge pull request #32416 from HanaBananaDev/HanaBananaDev-patch-1

Adding gymnasiumhilversum

philipto committed:
Merge branch 'master' into master

philipto committed:
Merge pull request #32453 from creativeriser/master

krmu

philipto committed:
Merge pull request #32650 from waterclear-bads/add-Vietnam-Japan-University

Pull request accepted.

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #32646 from sonbui00/hactech

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

Commit:
stcecilia

mithungawas committed:
Added dempocollege

RohitNalapur committed:
Create .gitkeep

RohitNalapur committed:
Create .gitkeep

RohitNalapur committed:
Add cmritonline.ac.in domain for CMR Institute of Technology

RohitNalapur committed:
Update cmritonline.txt

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #32666 from mithungawas/master

Pull request accepted.

RohitNalapur committed:
Delete lib/domains/in/ac/.gitkeep

Removed unwanted files

RohitNalapur committed:
Delete lib/domains/in/gitkeep.txt

Removed unwanted files

Rishi-Venkat committed:
Create rmd.txt

Rishi-Venkat committed:
Update rmd.txt

damilolasodiq committed:
Add university of guelph-humber

sirMikeee committed:
Add stcecilia.edu.ph domain for St. Cecilia's College - Cebu

philipto committed:
Merge pull request #32695 from sirMikeee/master

Pull request accepted.

philipto committed:
Merge pull request #32693 from damilolasodiq/master

Pull request accepted.

philipto committed:
Merge pull request #32682 from Rishi-Venkat/master

Pull request accepted.

philipto committed:
Merge pull request #32673 from RohitNalapur/master

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

Commit:
new file:   lib/domains/nl/yonder.txt

philipto committed:
Merge pull request #32474 from scentis/master

ucnl.edu.mx

philipto committed:
Merge pull request #32477 from 0xviny/master

My School Domain

philipto committed:
Rename GPUTC.txt to gputc.txt

philipto committed:
Merge pull request #32488 from EthanCoderPenguin2012/master

Added my school

philipto committed:
Merge pull request #32512 from SistemasSEAUMN/master

Create crg.txt

philipto committed:
Merge pull request #32703 from Yonder-ICT-Mediatechnologie/master

Pull request accepted.

kavyaNamballa committed:
Create rguktsklm.txt

DavidJustice committed:
Add JHN file

shub-o-glitch committed:
Create atharvacoe.txt

Commit:
Add Nihon University

KimgCheese committed:
Add files via upload

Added LCGS Domain

philipto committed:
Merge pull request #32729 from KimgCheese/master

Pull request accepted.

philipto committed:
Merge pull request #32721 from edo0159/master

Pull request accepted.

philipto committed:
Merge pull request #32715 from shub-o-glitch/master

Pull request accepted.

philipto committed:
Merge pull request #32711 from SaintJohnHenryNewman/master

Pull request accepted.

umais2005 committed:
Update IBA name to include location and abbreviation

philipto committed:
Merge pull request #32709 from kavyaNamballa/master

Pull request accepted.

trexment committed:
Salesianos Los Boscos La Rioja

danieljj28 committed:
Create xtart

UNF-CA committed:
Create unfc.txt

UNF-CA committed:
Merge branch 'JetBrains:master' into master

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #32740 from danieljj28/master

Pull request accepted.

philipto committed:
Merge pull request #32737 from trexment/master

Pull request accepted.

philipto committed:
Merge pull request #32731 from umais2005/master

Pull request accepted.

qshin34 committed:
Add files via upload

Deep3123 committed:
Added file for ldce college

philipto committed:
Merge pull request #32753 from Deep3123/master

Pull request accepted.

philipto committed:
Merge pull request #32752 from qshin34/master

Pull request accepted.

sahiljb committed:
Added file for RK University

wheisz-hub committed:
Added file for domain of HTL Trenkwalderstrasse

philipto committed:
Merge pull request #32774 from wheisz-hub/master

Pull request accepted.

philipto committed:
Merge pull request #32768 from sahiljb/master

Pull request accepted.

Skymmel committed:
Add files via upload

profluisotavio committed:
Add files via upload

yadavlavi7014 committed:
Add miet

umais2005 committed:
specifically unblock @khi.iba.edu.pk for iba.edu.pk domain. it is still unblocked.

BhavinPGitHub committed:
Create cranford.txt

Added Cranford

philipto committed:
Fraud removals and correct additions

philipto committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	lib/domains/pk/edu/iba.txt

Witty-chitwit committed:
Add files via upload

philipto committed:
Merge pull request #32803 from Witty-chitwit/master

Pull request accepted.

philipto committed:
Merge pull request #32792 from BhavinPGitHub/master

Pull request accepted.

philipto committed:
Merge pull request #32784 from umais2005/master

Pull request accepted.

philipto committed:
Merge pull request #32782 from yadavlavi7014/master

Pull request accepted.

philipto committed:
Merge pull request #32781 from profluisotavio/master

Pull request accepted.

philipto committed:
Merge pull request #32779 from Skymmel/master

Pull request accepted.

xenro45 committed:
Create nilesk12.txt

philipto committed:
Merge pull request #32806 from xenro45/master

Pull request accepted.

Delwer3 committed:
Create ctmcampusvirtual.txt

Add ctmcampusvirtual.com domain

charan862 committed:
Create lncharter/student.txt

shaikhAbuzar committed:
[India] Addition of MHSSCE

Addition of an Indian Undergraduate Education Institute named MH Saboo Siddik College of Engineering.

quanghuy1288 committed:
Create tnue.txt

quanghuy1288 committed:
Merge pull request #1 from quanghuy1288/quanghuy1288-patch-1

Create tnue.txt

10mca073 committed:
Added my college SRIMCA

Added college Shrimad Rajchandra Institute of Management and Computer Application (SRIMCA)

philipto committed:
Merge pull request #32830 from 10mca073/master

Pull request accepted.

philipto committed:
Merge pull request #32823 from quanghuy1288/master

Pull request accepted.

philipto committed:
Merge pull request #32821 from shaikhAbuzar/shaikhAbuzar-mhssce

Pull request accepted.

philipto committed:
Merge pull request #32820 from charan862/master

Pull request accepted.

philipto committed:
Merge pull request #32816 from Delwer3/master

Pull request accepted.

KazuMat committed:
Added new domain

philipto committed:
Niles West High School

philipto committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	lib/domains/org/nilesk12.txt
#	lib/domains/pk/edu/iba.txt

philipto committed:
Fraud

philipto committed:
Syntra-MVL, Gent

Peter-Akos committed:
Create esmt.txt

philipto committed:
Istituto Tecnico Industriale Pino Hensemberger

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #32844 from UNF-CA/master

Pull request accepted.

philipto committed:
Merge pull request #32842 from Peter-Akos/master

Pull request accepted.

philipto committed:
Merge pull request #32838 from KazuMat/master

Pull request accepted.

alanvitor-university committed:
Add unilavras.txt with university information

Please add UNILAVRAS as an educational university so that students and teachers can have access to the Jetbras suite of tools.

Satyam-Private committed:
Add files via upload

dhakarwalanil committed:
Add generated file: IIM Calcutta Added

AzureStar123 committed:
Add Iqra International School

Official Website: https://iisb.co
Higher Education Course (See IAL section): https://iisb.co/about-courses/

philipto committed:
Merge pull request #32867 from AzureStar123/master

Pull request accepted.

philipto committed:
Merge pull request #32855 from dhakarwalanil/master

Pull request accepted.

philipto committed:
Merge pull request #32853 from Satyam-Private/master

Pull request accepted.

philipto committed:
Merge pull request #32848 from alanvitor-university/unilavras-patch-1

Pull request accepted.

Scorpion7162 committed:
Add University of Birmingham School domain

shubhamdebroy committed:
Create kzu.txt

Add Kaziranga University. I gave proof in the request.

fluffy-takagi committed:
Add Tokyo Technical College

Codiosityy committed:
Add files via upload

philipto committed:
Merge pull request #32880 from Codiosityy/master

Pull request accepted.

philipto committed:
Merge pull request #32875 from shubhamdebroy/shubhamdebroy-patch-1

Pull request accepted.

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #32873 from Scorpion7162/master

Pull request accepted.

saifbashar committed:
Added Jamalpur Science & Technology University 

Jamalpur Science and Technology University (Bengali: জামালপুর বিজ্ঞান ও প্রযুক্তি বিশ্ববিদ্যালয়) is a government financed public university of Bangladesh. It was established on 28 November 2017 by the Act No. 24 of 2017 as the 44th public university of Bangladesh. Later it was renamed in 2024 as Jamalpur Science and Technology University. 
Website: https://jstu.ac.bd/
Wikipedia: https://en.wikipedia.org/wiki/Jamalpur_Science_and_Technology_University

saifbashar committed:
Merge pull request #1 from saifbashar/saifbashar-patch-1

Added Jamalpur Science & Technology University

BubbaJoeX committed:
North Iowa Area Community College

philipto committed:
Merge pull request #32887 from fluffy-takagi/master

Pull request accepted.

philipto committed:
Merge pull request #32885 from BubbaJoeX/master

Pull request accepted.

philipto committed:
Merge pull request #32881 from saifbashar/master

Pull request accepted.

tsuda-99 committed:
Create stu.txt

liang1481624299 committed:
feat: add Shunde Wonder School

omwork2005-cyber commented:
[coeptech.txt](https://github.com/user-attachments/files/22319254/coeptech.txt)
University URL https://www.coeptech.ac.in/

philipto committed:
Merge pull request #32899 from liang1481624299/master

Pull request accepted.

philipto committed:
Merge pull request #32898 from tsuda-99/master

Pull request accepted.

leereilly commented:
This repo has been archived, so I’ll be closing this out. I’m really grateful you took the time to share and be part of this project <3
